Task_HelloWorld/index.html

```diff
@@ -10,7 +10,7 @@
       <a href="../index.html">Go Back</a>
     </div>

```

Comment:
The opening and closing 'p' tags should be on separate lines for better readability.